examples/8a_benchmark_write_parallel.cpp

```diff
@@ -1131,6 +1150,13 @@ unsigned int AbstractPattern::getNumBlocks()
     return m_InRankMeshLayout.size();
 }
 
+inline openPMD::Iteration
+AbstractPattern::GetIteration(Series &series, int const iteration) const
```

Comment:
Do you still need that helper function now?

Reply:
It won't hurt to leave it there

Reply:
It's unnecessary verbose and might confuse users. This is an example, after all